draft-specifications/CapTP Specification.md

```diff
@@ -47,11 +47,11 @@ which together build up OCapN (Object Capability Network) specifications.
 
 This specification uses the following other specifications:
 
--   [Syrup](): The serialization format used for all messages between actors
+-   [Syrup](https://github.com/ocapn/syrup): The serialization format used for all messages between actors
```

Comment:
Might be worth linking our specific subset in Notation.md (or perhaps Notation.md is confusing and should be cut.)